packages/api/package.json

```diff
@@ -56,17 +56,11 @@
   "scripts": {
     "clean": "rm -rf lib && rm -f *.tsbuildinfo",
     "build": "tsc -p tsconfig.build.json",
-    "build:typedocs": "typedoc --exclude src/index.ts --out typedocs src",
-    "build:release": "yarn clean && yarn run build && yarn run build:typedocs",
     "check-build": "node -e \"(async function() { await import('./lib/index.js') })()\"",
     "check-types": "tsc",
-    "coverage": "codecov -F lodestar-api",
     "lint": "eslint --color --ext .ts src/ test/",
-    "lint:fix": "yarn run lint --fix",
-    "pretest": "yarn run check-types",
-    "test": "yarn test:unit && yarn test:e2e",
     "test:unit": "nyc --cache-dir .nyc_output/.cache -e .ts mocha 'test/unit/**/*.test.ts'",
-    "check-readme": "typescript-docs-verifier"
```

Comment:
CI only items like `check-readme`, `coverage` and `build:typedocs` and handled by other scripts only meant for CI